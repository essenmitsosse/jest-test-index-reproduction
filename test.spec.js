test.each(['A', 'B', 'C'])(
    'This is working: %#',
    () => expect(true).toMatchSnapshot()
)
test.each([{ NAME: 'A' }, { NAME: 'B' }, { NAME: 'C' }])(
    'This is not working: %#',
    () => expect(true).toMatchSnapshot()
)