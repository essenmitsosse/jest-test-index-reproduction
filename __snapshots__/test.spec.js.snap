// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is not working: %# 1`] = `true`;

exports[`This is not working: %# 2`] = `true`;

exports[`This is not working: %# 3`] = `true`;

exports[`This is working: 0 1`] = `true`;

exports[`This is working: 1 1`] = `true`;

exports[`This is working: 2 1`] = `true`;
